packages/frontend/src/components/ui/navbar/index.tsx

```diff
@@ -58,7 +58,7 @@ export const Navbar = ({
                 links={links}
                 onNavbarClose={() => setOpen(false)}
             />
-            <div className="flex justify-center px-4 xl:px-32 bg-transparent">
+            <div className="flex justify-center px-4 xl:px-32 bg-orange">
```

Comment:
I'm not sure we can do this as the creation form has a green bg for example and the navbar has to match. Previously there was a prop to the navbar that dictated the bg color which I removed. At this point I'd restore it.

Reply:
Fixed in 0a2477eb92e01e1cc5dc6c1be6be4d3d46d1f46c.